### ISSUE TITLE
Draw(w,h) was not producing a bitmap of the required size

### ISSUE THREAD

**Issue by the8thavatar** [vvvv/SVG#277]
I thought that the objective for this was to produce a bitmap of the desired size containing the image at 1:1 scaling but found that this was not the case.

I created DrawAtSize accepting width and height and using them in the Bitmap constructor and invoked DrawAtStdSize which did as Draw did with a ScaleTransform(1, 1);

**Comment by the8thavatar** [comment]
I had used an svg produced by System Architect which had this in the svg element
width="100%" height="100%" viewBox="75 33 1042 897" preserveAspectRatio="xMinYMin meet"
In comparison, an svg from Visio created by saving an individual symbol to svg worked with the original Draw(w,h) function whereas the function I created does not scale the image to the new extents.
The visio svg contained this in the svg element
width="0.720833in" height="1.32153in"
		viewBox="0 0 51.9 95.15" xml:space="preserve"

**Comment by Alko89** [comment]
I had the same issue drawing the bitmap in PictureBox. I ended up using this:
```
        svgDoc.Width = New SvgUnit(SvgUnitType.Pixel, pbSVG.Width)
        svgDoc.Height = New SvgUnit(SvgUnitType.Pixel, pbSVG.Height)
        pbSVG.Image = svgDoc.Draw()
```
This worked for my purpose.

**Comment by the8thavatar** [comment]
I solved it in the manner I described, the code is attached
[code excerpt.txt](https://github.com/vvvv/SVG/files/2694648/code.excerpt.txt)

Sample data attached. The test.svg is what is being rendered, the pic2.png is what Draw(270,232) gives me and pic1.png is what I obtained with the alternative approach. The shaded background covers the area the bitmap was supposed to cover, and the grid-background is a measurement indicator.
dmCapability.svg is an svg which did not give this problem.
[files.zip](https://github.com/vvvv/SVG/files/2694615/files.zip)
